src/orquestra/sdk/mlflow/_connection_utils.py

```diff
@@ -22,15 +139,95 @@ def get_temp_artifacts_dir() -> Path:
     do not need adjusting between runtimes.
     """
 
-    path: Path
-    if "ORQ_MLFLOW_ARTIFACTS_DIR" in os.environ:
-        # In Studio and CE there is an environment variable that points to the artifact
-        # directory.
-        path = Path(os.environ["ORQ_MLFLOW_ARTIFACTS_DIR"])
+    temp_dir_path: Path = Path(
+        os.getenv(_env.MLFLOW_ARTIFACTS_DIR) or DEFAULT_TEMP_ARTIFACTS_DIR
+    )
+    # In Studio and CE there is an environment variable that points to the artifact
+    # directory. If the artifact dir envvar doesn't exist, assume we're executing
+    # locally and use the default artifacts dir.
+
+    temp_dir_path.mkdir(parents=True, exist_ok=True)
+
+    return temp_dir_path
+
+
+def get_tracking_uri(workspace_id: str, config_name: Optional[str] = None) -> str:
+    """
+    Get the MLFlow tracking URI for the specified workspace.
```

Comment:
**nitpick**: if it exists, a docstring should convey more information than what can be read from the signature alone. WDYT?
```suggestion
    Infer a URI for accessing an MLflow tracking server deployed within this workspace.
    
    When run within an Orquestra cluster, this function returns an "internal URI" that helps save cluster bandwidth. This works even without specifying ``config_name``.
    
    When run outside a cluster, this function returns an "external URI". Requires passing ``config_name``.
```